js/app/modules/card/knowledgeDocument.js

```diff
@@ -219,6 +221,35 @@ const KnowledgeDocument = new Module.Class({
                     height_request: this.MIN_CONTENT_HEIGHT,
                     visible: true,
                 });
+
+                this.content_view.connect('handle-external-link', (view, uri) => {
+                    if (GLib.uri_parse_scheme (uri) !== 'file')
+                        return false;
+
+                    let mediabin = new EosKnowledgePrivate.MediaBin({
+                        audioMode: true,
+                        uri: uri,
+                    });
+                    mediabin.show();
+
+                    let lightbox = new Lightbox.Lightbox({
```

Comment:
Would it be possible to use the existing MediaLightbox ?

I am not sure it is, considering the audio files are not ObjectModels (and don't go through the HistoryStore), but one thing I noticed is that we can open this new Lightbox and it would only cover the document area, then a user could click on something else and the lightbox would be inaccessible (still playing the audio).